docs/pull_request_template.md

```diff
@@ -6,3 +6,4 @@
 - [ ] OpenAPI schema is updated (if applicable)
 - [ ] DB schema change has corresponding DB migration (if applicable)
 - [ ] README updated (if worker configuration changed, or if applicable)
+- [ ] Draft release notes have been updated
```

Comment:
Should we mention that this is to be done just before / after merging?

Reply:
Yeah, I think that makes sense. How about "Draft release notes are updated before merging"?